<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KBrokeragePage</class>
 <widget class="QWidget" name="KBrokeragePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>398</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="margin">
    <number>10</number>
   </property>
   <item>
    <widget class="QLabel" name="qLabel14">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Does this investment account have a brokerage account?&lt;p&gt;

&lt;b&gt;Check&lt;/b&gt; the mark if this investment account has an associated account for money that is not invested.&lt;br&gt;

&lt;b&gt;Uncheck&lt;/b&gt; it if you do not use this account for active stock brokerage. You should also uncheck it if the account is not maintained by a bank or broker.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_createBrokerageButton">
     <property name="text">
      <string>Create brokerage account</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="m_currencyFrame">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="textLabel1">
          <property name="text">
           <string>Currency</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="KMyMoneySecuritySelector" name="m_brokerageCurrency">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="m_accountNumberLabel">
          <property name="text">
           <string>Account number</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="KLineEdit" name="m_accountNumber"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="m_ibanLabel">
          <property name="text">
           <string>IBAN</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="KLineEdit" name="m_iban"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="spacer2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="spacer20">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>21</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KMyMoneySecuritySelector</class>
   <extends>KComboBox</extends>
   <header>kmymoneycurrencyselector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_createBrokerageButton</tabstop>
  <tabstop>m_brokerageCurrency</tabstop>
  <tabstop>m_accountNumber</tabstop>
  <tabstop>m_iban</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_createBrokerageButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_currencyFrame</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
